CesiumGltf/src/AccessorUtility.cpp

```diff
@@ -64,12 +59,55 @@ FeatureIdAccessorType getFeatureIdAccessorView(
     return AccessorView<int16_t>(model, *pAccessor);
   case Accessor::ComponentType::UNSIGNED_SHORT:
     return AccessorView<uint16_t>(model, *pAccessor);
+  // UNSIGNED_INT isn't supported for vertex attribute accessors, as described
+  // in the implementation notes for EXT_mesh_features. But neither
+  // EXT_instance_features nor EXT_mesh_gpu_instancing mention such a
+  // restriction.
```

Comment:
Nitpick, but do you mind moving this comment underneath the `case Accessor::ComponentType::UNSIGNED_INT` ?